src/transformers/optimization.py

```diff
@@ -32,6 +32,21 @@
 logger = logging.get_logger(__name__)
 
 
+def get_constant_lambda(_=None):
+    """
+    Return 1, independent from args.
+
+    Args:
+        _ ( *optional*, defaults to None):
+            Placeholder to argument, used to consistency with args in LambdaLR
+    
+    Return:
+        1 : int - constant lambda for constant scheduler
+    """
```

Comment:
No need for a documentation for a private function.

Reply:
Delete it.